Library/Homebrew/brew.sh

```diff
@@ -816,6 +816,22 @@ then
   unset HOMEBREW_RUBY_WARNINGS
 fi
 
+# Check for commands that should call `brew update --auto-update` first.
+AUTO_UPDATE_COMMANDS=(
+  install
+  outdated
+  upgrade
+  bump-formula-pr
+  bump-cask-pr
+  bundle
+  release
+)
```

Comment:
I'm guessing you want to remove the copy further up too.

Reply:
@Bo98 Whoops, yep, good catch.